### PR TITLE
i18n: Adds Occitan language

### PR DESCRIPTION
**Pull request by Mejans**
Hello!
My small contribution.

### CODE REVIEW

File: po/oc.po [hunk 1]
```diff
@@ -0,0 +1,626 @@
...
+msgid "Edit properties"
+msgstr "Modificar las proprietats"
+
+#: pdfarranger/metadata.py:212
+msgid "Property"
+msgstr "Proprietat"
+
+#: pdfarranger/metadata.py:212
+msgid "Value"
+msgstr "Valor "
```
Comment (dreua) [inline po/oc.po:201]: Here is an extra space: Typo?

File: po/oc.po [hunk 2]
```diff
@@ -0,0 +1,626 @@
...
+msgid "_Close"
+msgstr "_Tampar"
+
+#: pdfarranger/pdfarranger.py:943
+msgid "Save changes to “{}” before closing?"
+msgstr ""
+
+#: pdfarranger/pdfarranger.py:946
+msgid "Save changes before closing?"
+msgstr ""
```
Comment (dreua) [inline po/oc.po:279]: This and the following 5 strings are missing translations which might be especially important so people can make the right decisions and not lose work.

**Review by dreua — CHANGES_REQUESTED** [review]

**Comment by dreua** [comment]
> Partial translation to start

There are some strings missing some less some more important, see my comment above.

I guess there is no harm in merging it. Its probably better than having no translation at all and falling back to English.

File: po/oc.po (hunk 2 above)
Comment (Mejans) [reply po/oc.po:279]: Yes, will make a more complete PR later then!

File: po/oc.po (hunk 1 above)
Comment (Mejans) [reply po/oc.po:201]: Oups yes typo here!

File: po/oc.po (hunk 2 above)
Comment (dreua) [reply po/oc.po:279]: Ideally just add commits to this PR i.e. to your main branch. Thanks!

**Comment by jeromerobert** [comment]
Mercés

**Pull request merged**